torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -243,7 +243,7 @@ def speculate_subgraph(
     except Unsupported as ex:
         msg = (
             f"speculate_subgraph: while introspecting {description}, we were unable "
-            f"to trace function `{f.get_name()}` into a single graph. This means "
+            f"to trace function of type {f} into a single graph. This means "
```

Comment:
What's the problem with the original code?

Reply:
`f` is not supposed to be a UserDefinedFunctionVariableTracker always. It could be any callable tracker. 

In the case of an error reported internally, `f` was `NNModuleVariable`. Happy to take suggestions and make it better.

Reply:
Is it possible to print the name if `f` is a Function, otherwise, print the type?

Reply:
Yes, just sent a commit.